### ISSUE TITLE
Cuts off if the popup is to the right

### ISSUE THREAD

**Issue by mimansajaiswal** [AlexStepanischev/WeatherDock#5]
<img width="270" alt="image" src="https://user-images.githubusercontent.com/6592607/210231981-e98cac39-ed53-47b7-af0b-de00b164d847.png">
Adding a picture here to explain. I absolutely love this app, but at the moment it gets cut off on the right if the weather pop up is to the right.

**Comment by micosch** [comment]
I've noticed the same. I have the app towards the right and half the pop-up gets cut off. Other than that, gorgeous app

**Comment by AlexStepanischev** [comment]
@mimansajaiswal , @micosch Thank you for reporting, will be fixed with next release.

**Comment by micosch** [comment]
Great…now that I have your attention, what does the wind speed m/s stand for? I’m used to km/h as a unit of measure. Does it stand for metres per second?

Cheers,
Michael Schmidt

**Comment by AlexStepanischev** [comment]
@micosch yes, meters per second, well, probably I need to do a preference, where user could select desired unit for wind speed
